tests/wsl/enable_systemd.pm

```diff
@@ -17,27 +17,38 @@ sub run {
 
     assert_screen(['windows_desktop', 'powershell-as-admin-window']);
     $self->open_powershell_as_admin if match_has_tag('windows_desktop');
-    # In openSUSE the WSL shell is not root, so there's need to run become_root
-    # in order to write in the /etc/wsl.conf file
+
+    # Check that systemd is not enabled by default.
+    $self->run_in_powershell(
+        cmd => '$port.WriteLine($(wsl /bin/bash -c "systemctl is-system-running"))',
+        code => sub {
+            die("Systemd is running by default...")
+              unless wait_serial("offline");
```

Comment:
Syntax sugar, feel free to ignore.

I would be easier to read `wait_serial('offline') || die('Systemd is running by default...')` in this particular context.